### ISSUE TITLE
Question about HEV test set

### ISSUE THREAD

**Issue by thisishale** [MoonBlvd/tad-IROS2019#39]
Hello, 

Thank you very much for your contribution. I was wondering about the test set for HEV. The features provided in the readme file for the HEV dataset are train and validation. However, in Table 2. the results for HEV(test) are provided. Is the validation set used as the test set in this case?
